[2022-04-22 09:27:40,356] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: cso_2295_ipo_2_cso.py.transfer_cso_org manual__2022-04-22T09:27:30.102855+00:00 [queued]>
[2022-04-22 09:27:40,373] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: cso_2295_ipo_2_cso.py.transfer_cso_org manual__2022-04-22T09:27:30.102855+00:00 [queued]>
[2022-04-22 09:27:40,373] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-04-22 09:27:40,373] {taskinstance.py:1244} INFO - Starting attempt 1 of 1
[2022-04-22 09:27:40,373] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2022-04-22 09:27:40,385] {taskinstance.py:1264} INFO - Executing <Task(Postgres2File): transfer_cso_org> on 2022-04-22 09:27:30.102855+00:00
[2022-04-22 09:27:40,392] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'cso_2295_ipo_2_cso.py', 'transfer_cso_org', 'manual__2022-04-22T09:27:30.102855+00:00', '--job-id', '984', '--raw', '--subdir', 'DAGS_FOLDER/cso_2295/cso_2295_ipo_2_cso.py', '--cfg-path', '/tmp/tmp0ups1l0p', '--error-file', '/tmp/tmp94trn1_d']
[2022-04-22 09:27:40,393] {standard_task_runner.py:77} INFO - Job 984: Subtask transfer_cso_org
[2022-04-22 09:27:40,395] {standard_task_runner.py:52} INFO - Started process 1473 to run task
[2022-04-22 09:27:40,440] {logging_mixin.py:109} INFO - Running <TaskInstance: cso_2295_ipo_2_cso.py.transfer_cso_org manual__2022-04-22T09:27:30.102855+00:00 [running]> on host 22e04adb6bfa
[2022-04-22 09:27:40,497] {taskinstance.py:1431} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=cso_2295_ipo_2_cso.py
AIRFLOW_CTX_TASK_ID=transfer_cso_org
AIRFLOW_CTX_EXECUTION_DATE=2022-04-22T09:27:30.102855+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-04-22T09:27:30.102855+00:00
[2022-04-22 09:27:40,504] {base.py:79} INFO - Using connection to: id: postgres_default. Host: postgres, Port: 5432, Schema: ***, Login: ***, Password: ***, extra: {}
[2022-04-22 09:27:40,534] {taskinstance.py:1282} INFO - Marking task as SUCCESS. dag_id=cso_2295_ipo_2_cso.py, task_id=transfer_cso_org, execution_date=20220422T092730, start_date=20220422T092740, end_date=20220422T092740
[2022-04-22 09:27:40,571] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-04-22 09:27:40,618] {base.py:79} INFO - Using connection to: id: postgres_default. Host: postgres, Port: 5432, Schema: ***, Login: ***, Password: ***, extra: {}
[2022-04-22 09:27:40,620] {dbapi.py:225} INFO - Running statement: UPDATE list_flow_log
                        SET q_source=2,
                        q_target=2, t_end=NOW(), status=2,
                        optional_key='transfer_cso_org'
                        where id = 90;, parameters: None
[2022-04-22 09:27:40,621] {dbapi.py:233} INFO - Rows affected: 1
[2022-04-22 09:27:40,637] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
